src/lib.rs

```diff
@@ -2,11 +2,20 @@ use std::hash::Hash;
 
 mod buckets;
 mod classic;
+mod const_generics;
```

Comment:
请在这里添加, 可以避免没有开启这个 feature 的用户不需要编译这个 mod
```suggestion
#[cfg(feature = "const_generics")]
mod const_generics;
```

Reply:
这里会有个问题就是在 vscode 里面查看 `const_generics` 目录下的代码的时候会出现以下错误：  
```
file not included in module tree
```
不知道是不是和 rust-analyzer 的配置有关系，这里先按照作者的意见修改了